updater/updater.go

```diff
@@ -85,8 +87,18 @@ func (updater *Updater) updateMultiAddress(ctx context.Context) {
 			Params:  params,
 		}
 
-		address := fmt.Sprintf("http://%s:%v", addrs[i].IP4(), addrs[i].Port()+1)
-		response, err := updater.client.SendRequest(queryCtx, address, request, nil)
+		addrParts := strings.Split(addrs[i].Value, ":")
+		if len(addrParts) != 2 {
+			updater.logger.Errorf("[updater] invalid address value=%v", addrs[i].Value)
+			return
+		}
+		port, err := strconv.Atoi(addrParts[1])
+		if err != nil {
+			updater.logger.Errorf("[updater] invalid port=%v", addrParts[1])
+			return
+		}
+		addrString := fmt.Sprintf("http://%s:%v", addrParts[0], port+1)
```

Comment:
You can simply prefix `http://` to the front of `Address.Value` and this will be better supported than the code snippet here.